dev_guide/pod_autoscaling.adoc

```diff
@@ -324,7 +324,7 @@ kubernetesMasterConfig:
   ...
   apiServerArguments:
     runtime-config:
-    - apis/autoscaling/v2beta1=true
+    - apis/autoscaling/v1=true
```

Comment:
this can't change -- it's specifically referring to the beta version (but the above `v2alpha1` needs to change to `v2beta1`, plus, I think that the beta version is enabled by default now, so this section isn't needed.

Reply:
@DirectXMan12 
Per @sjenning in originating issue:
" All references to "autoscaling/v2beta1" should be updated to "autoscaling/v1" for 3.9 and later."
https://github.com/openshift/openshift-docs/issues/10074#issuecomment-401102988

Is this not true?

Reply:
Related: https://github.com/openshift/openshift-docs/issues/10737 ??

_"I've tried to create an Autoscaler object using the first definition, and I just get the error:
error: unable to recognize "create_tasksapp_autoscaler.yaml": no matches for extensions/, Kind=HorizontalPodAutoscaler_

_If I use oc autoscale, the created object does not use the v2 Autoscaler as specified in the docs. It creates the following object instead:_

_apiVersion: autoscaling/v1_

Reply:
I'm not sure we do a good communicating this, *but* the object is available in both forms, since it's a normal API object.  You just have to fetch it in the correct version (`oc get hpa.v2beta1.autoscaling`).  That being said *most* of the references should probably say `autoscaling/v1`, since it's still the stable version, and then the section discussing the new version should say `autoscaling/v2beta1` (since that's the new version).  I think @sjenning may have misread.